.github/workflows/reuse.yml

```diff
@@ -0,0 +1,32 @@
+---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+name: Verify REUSE
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  # Run CI once per day (at 04:45 UTC)
+  schedule:
+    - cron: '45 4 * * *'
+
+jobs:
+  check:
+    permissions:
+      contents: read
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          pip install reuse
+
+      - name: Check REUSE compliance (currently optional)
+        run: |
+          reuse lint || true
```

Comment:
Why not `rm` the files we're skipping like we previously did for the changelog fragments?

Reply:
Good point, improved that in 7eddfda7f850d17a6f90862ce061a74d1fb364f7.